demos/unity_states/cs/src/BallContract.cs

```diff
@@ -37,7 +37,7 @@ public ContractDataHandle Handle
         get
         {
             var p = new Ptr<ContractDataHandle>(ffi.BallContract_handle(this.Inner.Value.p));
-            return new ContractDataHandle(p);
+            return new ContractDataHandle(p, OwnershipSemantics.SharedRef);
```

Comment:
should be `Owned`: https://github.com/0xTELEPORTAL/constellation/blob/2f90dae624ebb56d001bc0f52674a2311c89149a/demos/unity_states/rust/src/lib.rs#L159